public/stylesheets/style.css

```diff
@@ -1,66 +1,211 @@
+<<<<<<< HEAD
 :root {
-    /* --primary-color: DARKMODE CHARCOL;
-    --secondary-color: BOX/ ELEMENT COLOR; */
-    --tertiary-color: (114, 216, 242);
+    --primary: #474343;
+    --secondary: #444950;
+    --tertiary: #9ae1e4d8;
+    --quatrinary: #6c6b67;
+    --light: #e3e4e3;
+    --dark: #1c2c25;
+    --error: rgb(228, 46, 1);
+    --border-radius: 0.3rem;
+  }
 
-*{
-    box sizing: border-box;
-}
-    body {
-        font-family: 'Love Light', cursive;
-        margin: 0;
-        display: flex;
-        flex-direction: column;
-        min-height: 100vh;
-        justify-content: flex-start;
-        background-color: var(--quatrianry);
-    }
-    a {
-        color: var (--dark);
-        text-decoration: none;
+  body {
+      font-family: 'Luxurious Roman', cursive;
```

Comment:
Is this the font (https://fonts.google.com/specimen/Luxurious+Roman) for the entire site or just for the title/headings? I feel that it might not be accessible in large blocks. Fonts like Roboto (https://fonts.google.com/specimen/Roboto) and Open Sans (https://fonts.google.com/specimen/Open+Sans?category=Sans+Serif) have higher accessibility scores than something without Serif styling.

Reply:
It should be for the entire site... I think I put it in the body... I will have to double check.

Reply:
The placement is right for sure! That bits perfection